cardano-node/src/Cardano/Node/Protocol/Cardano.hs

```diff
@@ -171,12 +172,17 @@ mkSomeConsensusProtocolCardano NodeByronProtocolConfiguration {
           -- version that this node will declare that it understands, when it
           -- is in the Mary era. Since Mary is currently the last known
           -- protocol version then this is also the Mary protocol version.
+          --
+          -- During testing of the Alonzo era, we conditionally declare that we
+          -- know about the Alonzo era. We do so only when a config option for
+          -- testing development/unstable eras is used. This lets us include
+          -- not-yet-ready eras in released node versions without mainnet nodes
+          -- prematurely advertising that they could hard fork into the new era.
           maryProtVer =
-            ProtVer 4 0
+            if npcTestEnableUnstableEras
```

Comment:
Is this to enable testing of a hardfork to Alonzo before consensus has merged their changes?

Reply:
Yes. Though I've now moved this patch into this PR instead: https://github.com/input-output-hk/cardano-node/pull/2400